common/presenters/move-to-card-component.js

```diff
@@ -30,6 +32,9 @@ function moveToCardComponent({
       href,
     })
 
+    // TODO: only include this is showTags is true???
```

Comment:
@teneightfive I'll double-check with Andy and Jason, but I'm guessing that if we're not showing the other tags, then the same applies for the important events too... Or am I overlooking all many of ph34rsome dashboard combos?

Reply:
Yes, I'd say that's correct. We use this option to hide tags from display's like the person lookup where we don't need to render tags.